store/rootmulti/store.go

```diff
@@ -626,6 +626,7 @@ func (rs *Store) PruneStores(clearPruningManager bool, pruningHeights []int64) (
 	}
 
 	rs.logger.Debug("pruning store", "heights", pruningHeights)
+	pruneHeight := pruningHeights[len(pruningHeights)-1]
```

Comment:
This is making the assumption that the last index pruneHeight is the largest height.